lib/sidekiq/logging.rb

```diff
@@ -47,7 +47,8 @@ def self.logger
     end
 
     def self.logger=(log)
-      @logger = (log ? log : Logger.new('/dev/null'))
+      null_path = (File.exists?('/dev/null') ? '/dev/null' : 'nul')
```

Comment:
The platform check gets a little verbose with the different types of platforms. New update removes boolean and simply calls File::NULL.